testing/watsonx-serving/prepare_scale.py

```diff
@@ -17,12 +18,19 @@ def compute_sutest_node_requirement():
     # the sutest Pods must fit in one machine.
     return min(user_count, machine_count)
 
+
+PSAP_ODS_SECRET_PATH = pathlib.Path(os.environ.get("PSAP_ODS_SECRET_PATH", "/env/PSAP_ODS_SECRET_PATH/not_set"))
+
 def prepare():
     """
     Prepares the cluster and the namespace for running the MCAD tests
```

Comment:
```suggestion
    Prepares the cluster and the namespace for running the Watsonx scale tests
```
(Don't need to fix in this PR)

Reply:
thanks @dagrayvid, applied !